### ISSUE TITLE
Glossary library in main.js is overriding src/glossary.js

### ISSUE THREAD

**Issue by smooth-opperator** [18F/glossary#17]

**Comment by afeld** [comment]
@kobekoto brought this up as well...this project is using [Browserify](http://browserify.org/) to compile the file from [`src/glossary.js`](https://github.com/18F/glossary/blob/master/src/glossary.js) to [`example/main.js`](https://github.com/18F/glossary/blob/master/example/main.js), which happens when you run `npm run build`.

https://github.com/18F/glossary/blob/0c06cf018516a1cfba9b8a6a1c27b2b711308074/package.json#L15

In other words, `main.js` shouldn't be edited directly. That being said, there is no mention of this! Interested in helping to document this?
- [ ] Add a note in [`CONTRIBUTING.md`](https://github.com/18F/glossary/blob/master/CONTRIBUTING.md) about the Browserify step
- [x] Have the browserify step add a note to `main.js` saying "this is a generated file, and should not be edited directly"

**Comment by mvattuone** [comment]
This is done @afeld

**Comment by afeld** [comment]
Hmm, I don't think so - still nothing in the docs that mention `npm build`.

**Comment by mvattuone** [comment]
Fair point, I'll include that where it mentions the compilation step.
